app/controllers/users.js

```diff
@@ -3,7 +3,6 @@ var cloud = require('../models/cloud');
 var crypto = require('../crypto');
```

Comment:
About the commit message.
"Remove knot service" / "Remove knot service DBus configuration"
On the description, you can say this service only set the user credentials.

Reply:
Done.